terminology/common-terminology.md

```diff
@@ -74,20 +74,32 @@ Various CCC projects will deliver code aimed at one or more of the packaging mod
 
 * Gramine could be utilized in the role of the “TEE Shim” across any package on SGX.
  
-## Package Deployability Prerequisites
+## Attestation
+
+However a workload is packaged, the resulting deployment should support interaction in a way that
+proves it is running in a TEE.
+
+While in traditional computing, there’s often a desire to authenticate a server, in Confidential
+Computing the goal is to authenticate the TEE, and attestation is the means to do that.
```

Comment:
I'm not sure whether "authenticate" is really the best word here, but let's discuss.

Reply:
@henkbirkholz please think about this and generate a pull request